ixmp/core.py

```diff
@@ -860,11 +858,13 @@ def equ(self, name, filters=None, **kwargs):
         return self.element('equ', name, filters, **kwargs)
 
     def clone(self, model=None, scen=None, annotation=None, keep_sol=True,
-              first_model_year=None):
+              first_model_year=None, platform=None):
         """clone the current scenario and return the new scenario
 
         Parameters
         ----------
+        platform : ixmp.Platform
```

Comment:
@zikolach can you move this docstring down to the bottom to match the function api?